utils/nctl/sh/utils/queries.sh

```diff
@@ -89,6 +89,17 @@ function get_chain_latest_block_hash()
         | sed -e 's/^"//' -e 's/"$//'
 }
 
+function get_chain_first_block_hash()
```

Comment:
I guess we don't actually need this function after all?

Reply:
I left it in, I may need it for something else down the line.